### DISCUSSION TITLE
Ungoogling the name?

### DISCUSSION THREAD

**Discussion by iskunk** [Eloston/ungoogled-chromium#2444] [Ideas]
### Description

Allow for a shorter application name, synonymous with "ungoogled-chromium" but without the word "google".

### Who's implementing?

- [X] I'm willing to implement this feature myself

### The problem

When the ungoogled-chromium browser is installed on the user's system, it (currently) does not make any attempt to identify itself as something distinct from regular Chromium. That is, the main executable is still named `chromium`, and while the Debian packaging does not add any GUI menu entries, the present approach would presumably be to add one for "Chromium" despite the different nature of the program.

In my effort to develop [tooling](https://github.com/ungoogled-software/ungoogled-chromium-debian/pull/328) to "ungoogle" a Debian Chromium package, I aimed to create ungoogled-chromium packages that can be installed concurrently with the regular chromium one. This required giving the application as installed a distinct identity.

The problem is, while "ungoogled-chromium" is a good name for the _project_---succinctly communicating its primary objective---it's somewhat awkward as a name for an _application_. Partly because it's a bit long... here is what it might look like in a Xubuntu application menu:

![Xubuntu screenshot](https://github.com/ungoogled-software/ungoogled-chromium/assets/523357/82041831-57fd-452d-a321-977dc0cad405)

The other reason is a bit more subjective. I love the idea of ungoogled-chromium, because it allows one to use the Chromium browser with all its arguable technical merits, without making the Faustian bargain of giving Google telemetry on everything. It enables a user to expunge Google from one particular (yet significant) aspect of their digital existence.

And yet, every time they reach for this privacy-enhancing tool, they are reminded of its relation to Google---even if the relationship is in the negative. They see the word "google" in the application name, and even though all the mysterious requests to 1e100.net et al. are gone, this one last callout to the behemoth remains. Which is all the more unfortunate when base Chromium doesn't prominently assert its Google identity, making it feel like the more "vanilla" experience even though under the hood it's anything but.

Or to put it another way: The name "ungoogled-chromium" may be thought of by some folks as synonymous with "unf\*\*\*ed-chromium". I won't begrudge anyone the sentiment. But even those folks may not want to see the word "f\*\*\*" put before them regularly, not least on occasions when they _aren't_ thinking about big-tech hegemony.

(My own views don't go quite that far, but I do find the name lacking in aesthetic appeal.)

### Possible solutions

I would like to propose the following alternative name for the application (not the project!), to be considered synonymous with "ungoogled-chromium" but without reference to the Goog:

**UnChromium**

This name is used by a [long-defunct Chromium-related project](https://github.com/amate/unChromium) that can be safely discounted. I've come across a [few](https://archived.moe/g/thread/62648144) [places](https://www.antixforum.com/forums/reply/38488/) [online](https://funnyjunk.com/The+more+you+know/funny-pictures/6206292/158) where this name has organically been used to refer to ungoogled-chromium, suggesting an intuitive connection between the two. [Searching for "unchromium" on DDG](https://duckduckgo.com/?t=ffab&q=unchromium&ia=web), amusingly, turns up ungoogled-chromium and related pages as the first several results. And I think it can be considered fitting that the difference between the two names is precisely the "googled" part.

I'm aware of the [earlier](https://github.com/ungoogled-software/ungoogled-chromium/issues/738) [discussions](https://github.com/ungoogled-software/ungoogled-chromium/issues/759) regarding the possibility of a name change. To be clear, this proposal has nothing to do with trademark legalities, only the subjective aspects of the name. (I'm assuming the legal concerns have been assuaged since the time those discussions took place.)

And I understand @Eloston's point about [not wanting to rebrand the browser](https://github.com/ungoogled-software/ungoogled-chromium/discussions/1756#discussioncomment-1743861)---since this is *just* Chromium minus the Google bits, not some whole new animal like Brave---and so deliberately went for a Chromium name tweak rather than throwing darts at a periodic table.

### Alternatives

I've thought of other possible names (e.g. UgChromium, UndChromium), but this is the best I've got. I'm not married to "UnChromium"; it could be something different that looks nice, and likewise elides Google. But for all the reasons given above, I think this one's going to be hard to beat.

### Additional context

If this proposal passes muster, then I would integrate the new name into the Debian package-conversion framework so that it can be seen by Debian users. I have at least two different contexts in mind:

1. The name visible in GUI menus, as in the example above;
2. The name of the .deb package, as seen by `apt-get`.

The first is the more important of the two, but consistency between the two is desirable. The "ungoogled-chromium" term would still show up in the package description and documentation, since those materials are oriented more toward "what is this?" and less toward day-to-day usage. Spelling out the precise relationship with Google in those contexts, I believe, remains necessary and appropriate.

How the new name is used elsewhere would be out of my hands, of course.

**Comment by PF4Public** [comment]
Please note that "Ungogled-Chromium Web Browser" is your invention. As far as I know, it goes under "Chromium" in other platforms. Only the version string page mentions it being ungoogled. And it is [by design](https://github.com/ungoogled-software/ungoogled-chromium#:~:text=ungoogled%2Dchromium%20retains%20the%20default%20Chromium%20experience%20as%20closely%20as%20possible.%20Unlike%20other%20Chromium%20forks%20that%20have%20their%20own%20visions%20of%20a%20web%20browser%2C%20ungoogled%2Dchromium%20is%20essentially%20a%20drop%2Din%20replacement%20for%20Chromium.) [so](https://github.com/ungoogled-software/ungoogled-chromium/pull/1332).

On the other hand [The Licence](https://github.com/ungoogled-software/ungoogled-chromium/blob/master/LICENSE) gives you broad freedom.

**Comment by iskunk** [comment]
> Please note that "Ungogled-Chromium Web Browser" is your invention. As far as I know, it goes under "Chromium" in other platforms. Only the version string page mentions it being ungoogled. And it is [by design](https://github.com/ungoogled-software/ungoogled-chromium#:~:text=ungoogled%2Dchromium%20retains%20the%20default%20Chromium%20experience%20as%20closely%20as%20possible.%20Unlike%20other%20Chromium%20forks%20that%20have%20their%20own%20visions%20of%20a%20web%20browser%2C%20ungoogled%2Dchromium%20is%20essentially%20a%20drop%2Din%20replacement%20for%20Chromium.) [so](https://github.com/ungoogled-software/ungoogled-chromium/pull/1332).

If both regular Chromium and ungoogled-chromium are installed, however, two "Chromium Web Browser" menu entries won't do.

> On the other hand [The Licence](https://github.com/ungoogled-software/ungoogled-chromium/blob/master/LICENSE) gives you broad freedom.

My immediate intent is to use this in the Debian packaging (as that's the angle I'm working from), so I think seeking some buy-in from the project itself is the right thing to do.

**Comment by PF4Public** [comment]
> If both regular Chromium and ungoogled-chromium are installed, however, two "Chromium Web Browser" menu entries won't do.

You should not be able to do so. They are mutually exclusive. They use the same profile after all!

**Comment by iskunk** [comment]
> You should not be able to do so. They are mutually exclusive.

Something I'd like to see happen, eventually, is ungoogled-chromium entering the Debian package repositories. This will not only make it available to Debian users as a whole (without needing to install external packages nor set up external repos), but also ensure that it is kept up-to-date and supported to the same degree that Debian supports the regular Chromium package.

(I'd like this to happen for other distros too, but Debian/Ubuntu is what I use, so I can only hope others will see this example as one worth following.)

This will be a non-starter, however, if ungoogled-chromium can't be installed together with the regular one. There are at least three objections I can think of to this:

1. Cannot accommodate multiple users on a system (e.g. a terminal server) who wish to use both;

2. Cannot accommodate a single user who would like to try out ungoogled-chromium, without giving up the regular one;

3. Does not allow alternate use cases for the "other" browser packages (e.g. someone may browse with ungoogled-chromium, but still want to use regular chromium-shell for some project).

If such a restriction were in place, one of the very first bug reports for the package would be to remove it. I addressed the issue proactively in my tooling because I knew it would be untenable otherwise.

> They use the same profile after all!

That's a whole 'nuther can of worms. Being able to install the two browsers side-by-side is different from actually _running_ them side-by-side. There are arguments for and against having ungoogled-chromium use the same config location as the regular one. I don't feel strongly either way about that, and haven't felt the need to put together a patch that changes it. But I'm not ruling out that the Debian folks may feel differently, and that such a patch may come about in the future, if only for their packaging.

**Comment by dmovztoq** [comment]
I wanted to open a ticket for suggesting despinning of the binary names, not necessairly for branding reasons but for avoiding crosstalk with base Chromium and Chrome, they all share "chrome.exe" executable and it has side effects, for example you can't run them all simultaneously out of the box.

If I launch Chromium first, every time I launch Ungoogled-Chromium the executable will spawn another Chromium window, and vice versa.

I'm currently looking at perhaps a workaround and a different solution as I imagine there might be one out there, likely involving forcing something in the command line.

I guess a lot of extra work might be necessary in order to rebraind/renaming the project and it's binaries to be completely separated? 

The core maintainers suggested that it is by design, that this project technically still recognizes it self as Chromium, but I haven't found an explanation as to why is that necessary? What is the technical reasoning behind it being strictly Chromium "just without google".

I also use Firefox a lot and then ofcourse Waterfox, and I like the Waterfox approach, but I'd imagine it has some downsides though I am not the most extreme of geek user exploring all sides of a browser so I'm not that familiar with ALL of the ups and downs, but it does use "Waterfox.exe" and has the separation done well and there's no issues with it running concurrently with Mozilla Firefox that I've ever noticed.
 
Doesn't mean UG has to be like that, diversity all the way, but for my use, Waterfox does do a lot of things what we wanted Firefox to be if it wasn't for Mozilla's politics.

Thanks
